README.md

```diff
@@ -1,7 +1,44 @@
 # pyCascadia
 Implementation of GEBCO cookbook remove-restore and other cleaning of topography/bathymetry. Uses `pyGMT`.
 
-## Shape file closing
+## Installation
+Some dependencies of the package are a bit fiddly to install, especially on Windows.
+We recommend to use a conda environment, and install the more difficult dependencies through `conda-forge`,
+by running the commands below in sequence:
+1. Clone this GitHub repository.
+1. In your local copy of the GitHub repository, create a conda environment named "cascadia" with Python 3.9 and install `pip`, `matplotlib`, `pytest`, and `xarray`.
```

Comment:
To a new user, it might seem the venv name *must* be "cascadia". We also might want to emphasize that Python 3.9 probably isn't exactly necessary.

```suggestion
1. In your local copy of the GitHub repository, create a conda environment with Python 3.9 and install `pip`. This will be used to install the rest of the dependencies. Other Python versions may also work however the code is actively tested with Python 3.9.
```